tests/unit/conftest.py

```diff
@@ -0,0 +1,35 @@
+"""Configuration for unit tests."""
+
+import pytest
+from httpx import AsyncClient
+
+
+@pytest.fixture
+def fake_post(monkeypatch):
```

Comment:
```suggestion
def mock_post(monkeypatch):
```
Renames from `fake` to `mock`. Other instances of this will also require renaming following this (e.g. in `test_instrument_conversion_service.py`)